File: 1351-count-negative-numbers-in-a-sorted-matrix/1351-count-negative-numbers-in-a-sorted-matrix.py
class Solution:
    def countNegatives(self, grid: List[List[int]]) -> int:
        res_count = 0
        for i in grid:
            for j in i[-1::-1]:
                if j >= 0:
                    break
                else:
                    res_count += 1
        return res_count
            

File: 1351-count-negative-numbers-in-a-sorted-matrix/1351-count-negative-numbers-in-a-sorted-matrix.go
func countNegatives(grid [][]int) int {
    count := 0
    for i:=0;i<len(grid);i++{
        fmt.Println(i)
        for j:=(len(grid[0])-1);j>=0;j--{
            fmt.Println(i,j)
            if grid[i][j] >= 0{
                break
            }else{
                count++
            }
        }
    }
    return count
}